.conf/dpv/protonvpn.template

```diff
@@ -0,0 +1,149 @@
+# ==============================================================================
+# Copyright (c) 2016-2020 Proton Technologies AG (Switzerland)
+# Email: contact@protonvpn.com
+#
+# The MIT License (MIT)
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR # OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+# ==============================================================================
+
+client
+dev tun
+proto $PROTOCOL
+
+remote $VPN_SERVER 443
+remote $VPN_SERVER 80
+remote $VPN_SERVER 1194
+remote $VPN_SERVER 5060
+remote $VPN_SERVER 4569
```

Comment:
@ravenclaw900 
I'm not happy with the fact that randomly five different ports are used here. This makes it unpredictable and limits the ability to control connections/requests via firewalls. Since 1194 is the default OpenVPN port, used by NordVPN configs as well, what do you think about using only that one? Mid-term, e.g. with next release, we could add another option to select the port (or even a multiple ones, if wanted by user), based on the known list of supported ports by that provider.

That also simplifies/hardens the killswitch rules accordingly.

Reply:
Sure, let's do that.  Indeed I did think about that when writing the killswitch, and the multiple ports can be a safety feature, especially when used with port 80.

Reply:
Changed:
- https://github.com/MichaIng/DietPi/commit/9b88c0e3df83de8b9b3be5b9923bacf56d2bda7b
- https://github.com/MichaIng/DietPi/commit/68312a96280745963f098f4cc460353e5009ec00

The "remote-random" settings actually has no meaning then, but let's keep it in case users modify the configs manually and since also the NordVPN configs (all with single 1194 remote only) have it as well 🤔. Plan for next release, aside of allowing to switch the port, is using a generic template for all providers, as NordVPN ones and our ProtonVPN template do now match nearly completely. Of course the server certs need to be appended per-provider. But probably we should add further providers first and see whether they either have very similar or matching configs as well, or at least work fine with them.